src/docs/get-started/install/_chromeos-android-sdk-setup.md

```diff
@@ -30,16 +30,14 @@ packages using the sdkmanager tool (version numbers here are
 the latest at time of publishing):
 
 ```terminal
-$ sdkmanager "build-tools;28.0.3" "emulator" "tools" "platform-tools" 
-"platforms;android-28" "extras;google;google_play_services" 
-"extras;google;webdriver" "system-images;android-28;google_apis_playstore;x86_64"
+$ sdkmanager "build-tools;28.0.3" "emulator" "tools" "platform-tools" "platforms;android-28" "extras;google;google_play_services" "extras;google;webdriver" "system-images;android-28;google_apis_playstore;x86_64"
```

Comment:
It seems unwise to have a line break in the middle of an argument.

Reply:
I think, i have removed it. There was one before